index.html

```diff
@@ -6,23 +6,27 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Hans Zizold'sPortfolio</title>
     <link rel="stylesheet" href="styles.css">
+    <script type="text/javascript" src="script.js"></script>
 </head>
 <body>
     <main class="main-container">
-        <header class="mobile-header">
-            <a href="#" id="name">Hans Paul Zizold</a>
-            <nav>
-                <button class="boton">
-                    <img src="images/menu.png" alt="Menu">
-                </button>
-                <ul class="menu">
-                    <li><a href="" class="options">Portfolio</a></li>
-                    <li><a href="" class="options">About</a></li>
-                    <li><a href="" class="options">contact</a></li>
-                    <li><img src="images/email.png" alt="email" class="img0"></li>
-                </ul>
-            </nav>            
-        </header>
+        <section class="mainheader">
+            <header class="mobile-header" id="mobheader">
+                <a href="#" id="name">Hans Paul Zizold</a>
+                <nav class="dropdown">
+                    <button onclick="dropmenu()" class="boton">
+                        <img src="images/menu.png" alt="Menu">
+                    </button>
+                </nav>            
+            </header>
+
+            <ul id="mymenu" class="menu">
+                <li class="lic"><a href="#" onclick="closemenu()" class="ex">x</a></li>
+                <li class="li1"><a href="#recent" class="options">Portfolio</a></li>
+                <li class="li1"><a href="#about" class="options">About</a></li>
+                <li class="li1"><a href="#always" class="options">Contact</a></li>
```

Comment:
When the user clicks (or taps) any option of the mobile menu, the mobile menu needs to disappear. Unfortunately, it's not happening in the case of this project.

https://www.loom.com/share/6cefd0c8bbba413fa1920d17d872b571

- [ ] Please, implement this feature too, so that after an option is selected, the mobile menu disappear.

Reply:
Hi @nicupop729, thank you for your correct observation. I think I corrected it.